test/stdlib/LibcFeatureFlag.swift

```diff
@@ -0,0 +1,33 @@
+// RUN: %target-typecheck-verify-swift
+// REQUIRES: executable_test
+// XFAIL: linux
+
+// This fails on Linux because _XOPEN_SOURCE or better is not defined
+
+#if os(OSX) || os(iOS) || os(watchOS) || os(tvOS)
+  import Darwin
+#elseif os(Linux) || os(FreeBSD) || os(PS4) || os(Android)
+  import Glibc
+#endif
+
+// drand48,mrand48,etc are only visible if _XOPEN_SOURCE (or better) is defined
+// on Glibc
+let a = drand48()
+print(a)
```

Comment:
…nor do we need to actually print anything. `_ = drand48()` is sufficient.

Reply:
yeah I wasn't it mattered since it was a %target-typecheck-verify-swift test. do we actually execute those?

Reply:
We don't (deliberately, to speed up testing time). Might as well not include things that aren't being tested.